services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/LicenseSpec.groovy

```diff
@@ -0,0 +1,29 @@
+package org.openkilda.functionaltests.spec.grpc
+
+
+import static org.openkilda.testing.ConstantsGrpc.SWITCH
+
+import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.grpc.speaker.model.LicenseDto
+import org.openkilda.messaging.error.MessageError
+
+import groovy.util.logging.Slf4j
+import org.springframework.http.HttpStatus
+import org.springframework.web.client.HttpClientErrorException
+import spock.lang.Narrative
+
+@Slf4j
+@Narrative("Test License.")
+class LicenseSpec extends BaseSpecification {
+    def "Not able to set incorrect license."() {
+        when: "Try to set incorrect license key."
+        String licenseFileName = "incorrectLicenseFileName.key"
+        String incorrectLicense = "incorrect license data"
+        grpc.setLicenseForSwitch(SWITCH, new LicenseDto(incorrectLicense, licenseFileName))
```

Comment:
Must be two tests:
1. Install license by "file name". File name is a file with license on switch.
2. Install license by "license data"

Reply:
Has been decided(together with Roman and Yaroslav)  to move these two tests into manual tests